src/web/routes.rs

```diff
@@ -10,9 +10,18 @@ pub(super) const DOC_RUST_LANG_ORG_REDIRECTS: &[&str] =
 pub(super) fn build_routes() -> Routes {
     let mut routes = Routes::new();
 
-    routes.static_resource("/robots.txt", super::sitemap::robots_txt_handler);
-    routes.static_resource("/sitemap.xml", super::sitemap::sitemap_handler);
-    routes.static_resource("/opensearch.xml", super::opensearch_xml_handler);
+    routes.static_resource("/robots.txt", PermanentRedirect("/-/static/robots.txt"));
+    routes.static_resource("/favicon.ico", PermanentRedirect("/-/static/favicon.ico"));
```

Comment:
Are you sure crawlers/browsers will follow this redirect? I guess there's no reason they wouldn't.

Reply:
[Seems like Google supports up to 5 redirections](https://developers.google.com/search/reference/robots_txt#handling-http-result-codes)

Reply:
Firefox and mobile Safari followed the favicon redirect in my testing.